.gitignore

```diff
@@ -7,6 +7,7 @@ pids
 lib-cov
 coverage
 .grunt
+.gitpod.yml
 dist/.local_storage*

```

Comment:
By curiosité, whats that ? :detective:

Reply:
So I can do some stuff on my Chromebook. Switched to Visual Studio Online btw which turns out to be way better. Highly recommend it! 👍

Reply:
I seee!
(I ended up there https://www.gitpod.io/)
Might give a try someday :)